### PR TITLE
Package matita.0.99.5

### PR DESCRIPTION
**Pull request by sacerdot**
### `matita.0.99.5`
An experimental, interactive theorem prover
Matita (that means pencil in italian) is an experimental, interactive theorem prover under development at the Computer Science Department of the University of Bologna.

---
* Homepage: http://matita.cs.unibo.it
* Source repo: git+https://github.com/sacerdot/matita.git
* Bug tracker: mailto:claudio.sacerdoticoen@unibo.it

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

**Comment by sacerdot** [comment]
There are some CI errors that I have an hard time understanding. In particular, 4.14 passed, but the "lower bounds" subtarget of 4.14 failed. Can somebody explain what the difference is?

**Comment by sacerdot** [comment]
The CI failures are all about other packages (dependencies) that fail on some exotic architectures:

- ocaml-expat fails for freebsd
- conf-gtksourceview3.0+2 fails for oraclelinux-9

File: packages/matita/matita.0.99.5/opam [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "An experimental, interactive theorem prover"
+description:
+  "Matita (that means pencil in italian) is an experimental, interactive theorem prover under development at the Computer Science Department of the University of Bologna."
+maintainer: ["Claudio Sacerdoti Coen"]
+authors: [
+  "HELM Team (Andrea Asperti, Ferruccio Guidi, Luca Padovani, Claudio Sacerdoti Coen, Enrico Tassi, Stefano Zacchiroli et alt.)"
```
Comment (raphael-proust) [inline packages/matita/matita.0.99.5/opam:8]:
```suggestion
  "HELM Team"
  "Andrea Asperti"
  "Ferruccio Guidi"
  "Luca Padovani"
  "Claudio Sacerdoti Coen"
  "Enrico Tassi"
  "Stefano Zacchiroli"
```

The format supports multiple authors so it makes sense to make use of it. If you want the "et al." you can maybe use "HELM Team (see <link>)"

**Comment by sacerdot** [comment]
@haochenx done, thank you!

**Review by haochenx — APPROVED** [review]
Looks all good to me!

**Comment by avsm** [comment]
Thanks! You may want to announce this on <https://discuss.ocaml.org>, where we have a Community category and an `announce` tag for this purpose.

**Pull request merged**